docs/guide/writing/support-team.rst

```diff
@@ -30,3 +37,34 @@ My advice: don’t be discouraged or alarmed if you get questions you can’t an
+
+Often this means that you have a huge backlog of tasks to juggle, and you cannot take on another role, like supporting client. However, this does not mean that you should lose your contact with this essential part of the business. Working as a technical writer within a support team is a great way to keep up with customer issues while maintain the documentation. All you need is a solid plan to make it happen.
+
+Consider these tips: 
+
+* Get the team on board
+* Make communication simple
+* Collaborate
+
+Get the team on board
+~~~~~~~~~~~~~~~~~~~~~
+
+It is always important to remember that when you work as a separate role within a team, it does not in fact make you apart from the team. Even though you write, and they support, you are still one team who are essential to each other. The better the documentation is, the less redundancies there are in tickets and calls coming in. This means that support will get less tickets, yes, but those tickets will contain challenging issues that really require their time and skills. This is a rewarding, symbiotic relationship. When they have meetings on issues, you attend; when you have doc issues, you communicate those with them. You can’t be a team unless you understand each other and the unique issues each of you face. 
+
+Make communication simple
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you are writing the documentation full-time, you have no time to answer calls or respond to tickets. You are relying on your team to let you know which issues are coming in often enough to create a doc page or FAQ question. However, they are busy too. Anyone with exposure to support work will know that it is tough job that requires helping as many customers as you can in a single day while maintaining the highest satisfaction rating possible. So, they need to be able to send you their issues quickly and efficiently. There are several ways to do this, creating a dedicated email account for doc tasks where they can send issues, creating a Slack command that assigns you the task automatically, having a team Trello board, and so on. Find the solution that works best for your team. And don’t despair – if one solution does not work, move on to the next.
+
+Collaborate
+~~~~~~~~~~~
+
+When a member of your team assigns you a doc issue, give them partial ownership of that issue. Even though you are writing the doc, they should be the one to review it to ensure that you are answering the issue correctly and there are no gaps. And once they do that, show gratitude. Ensure your team know how much the collaborative relationship is valuable to the documentation. Small things, like showing how many hits an article that was created from an assigned issue gets per month will help your team understand how valuable their contributions are, and it will encourage even more contributions. 
```

Comment:
* Change "Ensure your team know" to "Ensure your team knows" because WTD is based on American English per verbal confirmation from @plaindocs.